internal/pkg/template/workload_test.go

```diff
@@ -314,40 +314,32 @@ func TestSecretsManagerName_ValueFrom(t *testing.T) {
 
 func TestWorkload_HealthCheckProtocol(t *testing.T) {
 	testCases := map[string]struct {
-		opts     WorkloadOpts
+		opts     ALBListenerRule
```

Comment:
Can we rename `func TestWorkload_HealthCheckProtocol(t *testing.T)` to `func TestALBListenerRule_HealthCheckProtocol(t *testing.T)`